iosHyperskillApp/iosHyperskillApp/Sources/Modules/Step/ViewData/StepViewDataMapper.swift

```diff
@@ -48,6 +48,6 @@ final class StepViewDataMapper {
             count: commentStatisticsEntry.totalCount
         )
 
-        return StepCommentStatisticViewData(id: commentStatisticsEntry.thread.name, title: title)
+        return StepCommentStatisticViewData(id: commentStatisticsEntry.thread?.name ?? title, title: title)
```

Comment:
If `commentStatisticsEntry.thread == nil` then we don't want to display this in the UI. It's better to add an early `guard` statement and return a `nil` value.

^^ If this will introduce a lot of changes in the current code, please let me know and we will think what to do.